src/roi/Roi.ts

```diff
@@ -608,3 +637,101 @@ function getBorders(roi: Roi): { ids: number[]; lengths: number[] } {
     lengths: borderLengths,
   };
 }
+
+function getEllipse(
+  roi: Roi,
+  options: { nbSD: number; scale: number },
```

Comment:
Please remove those options. The expected current behaviour is to return the ellipse with the same surface.